bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/fixity/FixityChecker.scala

```diff
@@ -81,7 +82,7 @@ trait FixityChecker[BagLocation <: Location] extends Logging {
   private def parseLocation(
     expectedFileFixity: ExpectedFileFixity
   ): Either[FileFixityCouldNotRead[BagLocation], BagLocation] =
-    locate(expectedFileFixity.uri) match {
+    expectedFileFixity.uri.locate match {
```

Comment:
👍

Good spot (I think the `locate()` method came in when I was disambiguating ObjectLocation, and I forgot to revert it).

How hard would it be to pull out into a separate patch? That would be mergeable immediately and make this diff simpler.

Reply:
yes, unfortunately the refactoring commits are a bit entangled... the last commit is entirely refactoring, but relies on other refactoring I did before... I think it might be a bit complicated now